webui/apps/docs/src/content/docs/docs/howto/communication.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Agent-Agent Communication
-description: References - Agent Communication
+title: Enable Agent Interactivity
```

Comment:
nit: it is always enabled, but this shows how to utilize it. Utilizing Agent Interactivity?